pkg/api/validation/validation.go

```diff
@@ -1426,28 +1427,62 @@ func ValidateLimitRange(limitRange *api.LimitRange) errs.ValidationErrorList {
 			q := limit.Default[k]
 			defaults[string(k)] = q.Value()
 		}
+		for k := range limit.DefaultRequest {
+			allErrs = append(allErrs, validateResourceName(string(k), fmt.Sprintf("spec.limits[%d].defaultRequest[%s]", i, k))...)
+			keys.Insert(string(k))
+			q := limit.DefaultRequest[k]
+			defaultRequests[string(k)] = q.Value()
+		}
+		for k := range limit.LimitRequestRatio {
+			allErrs = append(allErrs, validateResourceName(string(k), fmt.Sprintf("spec.limits[%d].limitRequestRatio[%s]", i, k))...)
+		}
 
 		for k := range keys {
 			minValue, minValueFound := min[k]
 			maxValue, maxValueFound := max[k]
 			defaultValue, defaultValueFound := defaults[k]
+			defaultRequestValue, defaultRequestValueFound := defaultRequests[k]
 
+			// max > min
```

Comment:
I think if you're referring to the condition that causes an error, the comment should say min > max.
If you're referring to the condition that is accepted, the comment should say max >= min.